CHANGELOG.rst

```diff
@@ -11,6 +11,28 @@
 Changelog
 +++++++++
 
+0.14.0
+======
+
+- Obey the ``--skip-subprojects`` option passed to the ``meson
+  install`` command.
+- Implement support for building wheels targerting the Python limited
+  API. Extension modules targeting the Python limited API can be
+  easily build starting with the upcomin Meson 1.3.0 release.
+- Instead than silently using the version declared in ``meson.build``,
```

Comment:
- instead than -> instead of
- perhaps turn it around: _"When `pyproject.toml` does not contain a `version` field and ..."_. Might be a little easier to understand for the reader whether this applies to them.